ExtLibs/ArduPilot/Mavlink/MAVLinkInterface.cs

```diff
@@ -2500,6 +2500,19 @@ public bool doHighLatency(bool onoff= false)
             return ans1;
         }
 
+        /// <summary>
+        /// Control ICE Engine
+        /// </summary>
+        /// <param name="onoff">true to enable, false to disable</param>
+        /// <returns></returns>
+        public bool doEngineControl(bool onoff = false)

```

Comment:
can we change to sysid and compid needed here.

im trying to make this class able to target any connected mav. feed the sysid and compid from the ui level

Reply:
Good call, sorry.

Done.